test/box/moved_delayed_free_mode.test.lua

```diff
@@ -0,0 +1,49 @@
+env = require('test_run')
+fiber = require('fiber')
+test_run = env.new()
+
+objcount = 10000
+test_run:cmd("setopt delimiter ';'")
+function test(op, count, space)
+    if op == 1 then
+        for key = 1, count do
+            space:insert({key, key + 1000})
+        end
+    elseif op == 2 then
+        for key = 1, count do
+            space:replace({key, key + 5000})
+        end
+    elseif op == 3 then
+        for key = 1, count do
+            space:upsert({key, key + 5000}, {{'=', 2, key + 10000}})
+        end
+    elseif op == 4 then
+        for key = 1, count do
+            space:delete({key})
+        end
+    else
+    	assert(0)
+    end
+end;
+test_run:cmd("setopt delimiter ''");
+
+space = box.schema.space.create('test')
+space:format({ {name = 'id', type = 'unsigned'}, \
+               {name = 'year', type = 'unsigned'} })
+_ = space:create_index('primary', { parts = {'id'} })
+channel = fiber.channel(1)
+
+-- Try to insert/replace/upsert/delete tuples in fiber
+-- in parallel with the snapshot creation.
```

Comment:
Actually I don't get the purpose of test. What are you verifying? Now it looks like check that Tarantool doesn't crash. I bet there's already test checking that parallel operations during snapshot are processed correctly.

Reply:
Yes it's check that Tarantool doesn't crash! Other tests check this not with a hundred percent guarantee.